HelpSource/Guides/WritingClasses.schelp

```diff
@@ -32,72 +37,147 @@ MyClass : SomeSuperclass {
 Without specifying a superclass, link::Classes/Object:: is assumed as the default superclass.
 
 code::
-MyClass { // :Object is implied
+MyClass { // : Object is implied
 
 }
 ::
 
+This is why in the above example, the message code::new:: can be called without being explicitly defined in the class.
+
+
+
 section:: Methods
 
-class methods are specified with the asterisk within the class method, the keyword code::this:: refers to the class. A
-class in SuperCollider is itself an object.  It is an instance of link::Classes/Class::. Instance methods are specified
-within the instance method, the keyword code::this:: refers to the instance.
+subsection:: Instance Methods
+
+Each object instance responds to its strong::instance methods::.Within instance methods, the keyword code::this:: refers to the instance itself.
+
+An instance is independent of all the other instances, unless otherwise specified.
+
 
 code::
 MyClass {
-    *classMethod { arg argument;
 
-    }
+	instanceMethod { | argument |
+		this.anotherInstanceMethod(argument)
+	}
 
-    instanceMethod { arg argument;
-
-    }
+	anotherInstanceMethod { | argument |
+		"hello instance".postln
+	}
 }
 ::
 
+This could then be used as follows:
+
+code::
+a = MyClass.new // returns a new instance
+a.instanceMethod // posts "hello instance"
+::
+
 
 To return from the method use teletype::^:: (caret). Multiple exit points also possible. If no teletype::^:: is
 specified, the method will return the instance (and in the case of Class methods, will return the class). There is no
 such thing as returning void in SuperCollider.
 
 code::
 MyClass {
-    someMethod {
-        ^returnObject
-    }
-
-    someOtherMethod { arg aBoolean;
-        if(aBoolean,{
-            ^someObject
-        },{
-            ^someOtherObject
-        })
-    }
+	someMethod {
+		^returnObject
+	}
+
+	someOtherMethod { | aBoolean |
+		if(aBoolean) {
+			^someObject
+		} {
+			^someOtherObject
+		}
+	}
+
+}
+::
+
+
+subsection:: Class Methods
+
+An object's strong::class methods:: are defined together with its instance methods. They are specified with the asterisk (teletype::*::).
+
+
+A link::Classes/Class:: is itself an object. It is what all instances of it have in common. Class methods are the instance methods of the object's class. That's why within class methods, the keyword code::this:: refers to the class.
```

Comment:
This is a good explanation! one question, would it be more accurate to say "instance methods of the object's metaclass"?